hack/helm/pre_helm.go

```diff
@@ -49,6 +51,13 @@ const annotationsDest = `  template:
       labels:
         control-plane: controller-manager`
 
+const envHTTPSSrc = `value: "433"`
+
+const envHTTPSDEST = `value: {{default 8443 .Values.ingressHTTPSPort |quote}}`
+const envHTTPSrc = `value: "8080"`
+
+const envHTTPDEST = `value: {{.Values.ingressHTTPPort |quote}}`
```

Comment:
nit: maybe we can clean this up a little?

```
const (
        envHTTPSSrc = `value: "433"`
        envHTTPSDEST = `value: {{ default 8443 .Values.ingressHTTPSPort | quote }}`
        envHTTPSrc = `value: "8080"`
        envHTTPDEST = `value: {{ .Values.ingressHTTPPort | quote }}`
)
```

Reply:
done 8b0052a